yarn.lock

```diff
@@ -4761,13 +4690,34 @@ __metadata:
   languageName: node
   linkType: hard
 
+"@types/d3-array@npm:^3.0.3":
```

Comment:
There are a bunch of duplicates I can't resolve here with d3. This is because of recharts and one of its dependency (victory) that have conflicting types deps